doc/lua_api.txt

```diff
@@ -6030,6 +6032,8 @@ object you are working with still exists.
     * Server-sent FOV value. Returns 0 if an FOV override doesn't exist.
     * Boolean indicating whether the FOV value is a multiplier.
     * Time (in seconds) taken for the FOV transition. Set by `set_fov`.
+* `get_lifetime()`: returns the elapsed time since the player joined the game.
+* `get_idletime()`: returns the elapsed time since the player was last active.
```

Comment:
`was last active`. From the engine's POV they're active as long the connection persists. I think the definition of "activity" should be left to mods to decide. For example, players could move even though they're not active (falling, drowning) either caused by themselves initially, or by players who let them fall.

Reply:
I feel as if that would be over-complicated matters. For purposes of this PR, "last active" is simply the receiving of new events from the client. Perhaps at some point filtering could be supported. But I think that is best left for a future PR. My goal is to provide at least the basic functionality that would be sufficient for most purposes.